locale/en/blog/release/v6.0.0.md

```diff
@@ -0,0 +1,172 @@
+---
+date: 2016-04-26T20:35:53.135Z
+version: 6.0.0
+category: release
+title: Node v6.0.0 (Current)
+slug: node-v6-0-0
+layout: blog-post.hbs
+author: James M Snell
+---
+
+It's here! Just three days shy of exactly six months after we released Node v5.0.0, the Node.js developers are happy to announce that Node.js v6 is available and ready for use! 
+
+This new major version of Node.js delivers significant improvements in performance, reliability, usability and security. Module loading is nearly four times faster than in Node.js v4, tests and documentation have seen significant improvements, usability of the Buffer and File System APIs has been improved, and Node.js now supports 93% of JavaScript language features thanks to V8 version 5.0.
+
+In accordance with our [Long Term Support plan](https://github.com/nodejs/LTS/), Node.js v4 'Argon' will continue to be supported as the *Active* Long Term Support stream until April of 2017, after which it will transition into what we called *Maintenance* LTS. The new Node.js v6 branch will become the new *Active* Long Term Support stream in October of this year, and will continue to be supported actively until April of 2018. If our LTS plan is new to you, [read more](https://medium.com/@nodesource/essential-steps-long-term-support-for-node-js-8ecf7514dbd#.z6bzdfogi) about how it works so that you can make a properly informed decision regarding which version of Node.js to adopt.
```

Comment:
> as the *Active* Long Term Support stream

nit: as **an** *Active* Long Term Support stream

There will be an overlap of Active LTS from October 2016 - April 2017